wdl/model/draft2/src/main/scala/wdl/draft2/model/WdlNamespace.scala

```diff
@@ -157,7 +157,14 @@ object WdlNamespace {
   }
 
   private def load(workflowSource: WorkflowSource, resource: String, importResolver: Seq[Draft2ImportResolver], importedAs: Option[String], root: Boolean = true): Try[WdlNamespace] = Try {
-    WdlNamespace(AstTools.getAst(workflowSource, resource), resource, workflowSource, importResolver, importedAs, root = root)
+    val maybeAst = Option(AstTools.getAst(workflowSource, resource))
```

Comment:
Old version crashed and burned with an empty file

Reply:
There were two distinct paths by which we crashed and burned with an empty file. This is the second of two, and was only exposed after fixing the first. I just happened to copy-paste the same comment in the PR